llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -745,9 +828,9 @@ class ModuleSplitter {
     ++GMapIt;
 
     std::unique_ptr<Module> SplitModule{nullptr};
-    if (IsSplit && !SplitModuleEntryPoints.empty())
+    if (IsSplit && !SplitModuleEntryPoints.empty()) {
       SplitModule = extractCallGraph(*InputModule, SplitModuleEntryPoints);
-    else {
+    } else {
```

Comment:
FYI: existing code is correct from [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements) point of view

Reply:
Yes, I get it but for me those idents in the end of every block where '}' is interchanged with no brace looks unclear, so this is why I've added the braces.